ogre2/src/Ogre2GpuRays.cc

```diff
@@ -713,7 +774,26 @@ void Ogre2GpuRays::Setup1stPass()
           static_cast<Ogre::CompositorPassSceneDef *>(
           colorTargetDef->addPass(Ogre::PASS_SCENE));
       // set camera custom visibility mask when rendering laser retro
-      passScene->mVisibilityMask = 0x01000000;
+      passScene->mVisibilityMask = 0x01000000 &
+          ~Ogre2ParticleEmitter::kParticleVisibilityFlags;
+    }
+
+    Ogre::CompositorTargetDef *particleTargetDef =
+        nodeDef->addTargetPass("particleTexture");
+    particleTargetDef->setNumPasses(2);
+    {
+      // clear pass
+      Ogre::CompositorPassClearDef *passClear =
+          static_cast<Ogre::CompositorPassClearDef *>(
+          particleTargetDef->addPass(Ogre::PASS_CLEAR));
+      passClear->mColourValue = Ogre::ColourValue(0, 0, 0, 0);
```

Comment:
Can we use `Ogre::ColourValue::Black`? Or does that not give us the alpha value we want?

Reply:
done. 4108632